State of the Art/Deep learning & Machine learning/MachineAndDeepLearning.md

```diff
@@ -0,0 +1,76 @@
+# AI, Machine Learning, Deep Learning
+
+How are thoses three things related? An image explains better than words:
+
+
+![error loading image](./images/AIMachineDeepGraphic.png)
+
+The field of Deep learning is included in the field of Machine learning, which is itself included in the field of Artificial Intelligence.
+
+
+## The first machine that learned
+
+The first machine invented that learned was made in 1957 and was called the Perceptron. A very simple simulated neuron with adaptive "synaptic neurons" which adapted with trianing.
+
+
+When we talk about Machine Learning we mainly talk about supervised learning models. Just like the Support Vector Machine (*SVM*) whose concept is as followed:
+
+> Input data is given a set class by the superviser (human)
+
+> When ever another input data is given, the SVM approximate its class depending on his training
+
+> SVM cannot estimate a class it doesn't know
+
+
+This model has been used for over 50 years and is still used nowadays.
+
+## Machine learning / Deep learning
+*If we take the exemple of image processing:*
+
+With a Machine learning approche:
+
+![error loading image](./images/MachineLearningFeaturesApproche.png)
+
+With a Deep learning approche:
+
+![error loading image](./images/DeepLearningFeaturesApproche.png)
+
+We can make a parralel of this with Deep learning in text analysis:
+
+Text --> Caracters --> Words --> Word Group --> Clause --> Sentense --> Story --> Classifier
+
+
+But how does it translate with a trainable AI?
+
+**Multi-Layer Neural Network.**
+
+
+![error loading image](./images/MultiLayerNeuralNetwork.png)
+
+As you can see in the image above, we use the image with different convolution mask on the image to extract some features from the image. These images are used by combination in a first neural layer which trains its "synaps" with a non linear function to interact with lower level layers.
+
+
+One of the most used Multilayer Neural Network is the one invented by Yan LECUN (and associates) in 1989 called LeNet. LeNet is what we now call a **ConvNet** *or* Convolution Neural Network (**CNN**).
+
+This model is now used everywhere. Image processing, text processing. Supervised models, unsupervised models.
+
+First AI could define the image it was show, then it could seperate different objects inside an image. In 2015 we could creat an AI that described what it was seeing. However this was on static images. We are now capable of convolution on video, enabling an AI to understand what is happening in a video.

```

Comment:
review this formulation : First AI could define the image it was show
create with "e" at the end